server/app/routes/index.js

```diff
@@ -25,10 +41,18 @@ app.use(
   cors(corsOptions),
 );
 
+app.use(jwtCheck);
```

Comment:
Let's no have the middlewares, the server setup and the router in the same file, it's just going to be a mess in the future.

i would add a file called `app.js` outside the routes folder, and in this file `routes/index.js` i would use the `express.Router()` method to make the router modular

Reply:
[click this link to know more about `express.Router()`](https://expressjs.com/en/guide/routing.html#express-router)

Reply:
I agree I'll fix that soon, so ill split all the routes to the router, then all middlewares and set up to another on the app.js